conf/schemas/osquery.json

```diff
@@ -18,6 +18,7 @@
       "optional_top_level_keys": [
         "decorations",
         "epoch",
+        "log_type",
```

Comment:
Is it `log_type` also added it `osquery:batch`?

Reply:
Good question let me double-check , I don't use batch myself, so it may need included in the JSON